client/analytics/components/leaderboard/index.js

```diff
@@ -50,13 +51,15 @@ export class Leaderboard extends Component {
 		const { isRequesting, isError, totalRows, title } = this.props;
 		const rows = this.getFormattedRows();
 
+		const classes = classnames( 'woocommerce-leaderboard', 'woocommerce-analytics__card' );
```

Comment:
Nitpick - this doesn't need to use `classnames` - it could just be a string literal.